### DISCUSSION TITLE
Nvidia RTX4090 and AMD 7900XTX benchmark

### DISCUSSION THREAD

**Discussion by Ryuukeisyou** [comfyanonymous/ComfyUI#2588] [General]
I made a simple benchmark test for the two gpus on my computers. Hope this can help anyone who is interested.

7900XTX is about **57%** of RTX4090 at low resolution and batchsize (512x512x1), 
yet only achieves **33%** at higher resolution and batchsize (768x768x4).
would get even lower to **29%** if RTX4090 uses xformers (tested but not listed below).

And so I wonder what would be the cause? Would anyone have any ideas?

Here are some details.

### Nvidia RTX4090
cpu: Ryzen9 7950x, 16core-32thread, 4.5GHz to 5.7GHz
mem: 64GB (32GB x 2) 6000MHz
os: Windows 11 23H2
pytorch:  2.1.2+cu121

### AMD 7900XTX
cpu: EPYC 7F52, 16core-32thread, 3.5GHz to 3.9GHz
mem: 64GB (32GB x 2) 2666MHz
os: Ubuntu 22.04
pytorch: 2.3.0.dev20240118+rocm5.7

### Common
checkpoint: v1.5-pruned-emaonly.safetensors
positive prompt: 1girl
steps: 100
sampler: euler
args: --dont-upcast-attention --use-pytorch-cross-attention
seed: run random seed  5 times to get average

### Results
average it/s
![avg it_s](https://github.com/comfyanonymous/ComfyUI/assets/33691801/9b003f77-4894-4185-8596-d65a1d1de603)
average pixels*it/s
![avg pixels_it_s](https://github.com/comfyanonymous/ComfyUI/assets/33691801/16c4db7b-2ab1-41af-b91f-c8b692ddfb9d)

**Comment by Dampfinchen** [comment]
How can I install Xformers in Comfy UI? I am using the standalone package. I've tried to install it via pip install but it still doesn't use it. Help would be appreciated.

**Reply by jbeougher7924** [reply]
There are so many reason why this will not give you accurate results.  Neither system config is the same.  Rerun your test with the same workstation but only change out the video card and see what the results are.  Have each GPU perform the same test at least 3 times and average the results.
